service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureStorageAccessService.java

```diff
@@ -78,6 +84,8 @@ public AzureStorageAccessService(
     this.storageAccountKeyProvider = storageAccountKeyProvider;
     this.azureConfiguration = azureConfiguration;
     this.workspaceService = workspaceService;
+    this.storageAccountCache =
+        new ConcurrentHashMap<>(new PassiveExpiringMap<>(1, TimeUnit.HOURS));
```

Comment:
the implementation of this`ConcurrentHashMap` constructor is
```
    public ConcurrentHashMap(Map<? extends K, ? extends V> m) {
        this.sizeCtl = DEFAULT_CAPACITY;
        putAll(m);
    }
```
The use of `PassiveExpiringMap` is ignored.